pyaedt/modeler/Primitives3D.py

```diff
@@ -562,7 +562,7 @@ def create_circle(self, cs_plane, position, radius, numSides=0, is_covered=True,
         return self._create_object(new_object_name)
 
     @aedt_exception_handler
-    def create_ellipse(self, cs_plane, position, major_raidus, ratio, is_covered=True, name=None, matname=None):
+    def create_ellipse(self, cs_plane, position, major_radius, ratio, is_covered=True, name=None, matname=None):
```

Comment:
@maxcapodi78 I don't know if we should add a deprecation warning as it is an argument name and not the method name that was changed. Let say that one customer uses named arguments in his script with something like:
`create_ellipse(, "XY", [0., 0.,0.], major_raidus = 3.2, ratio = 0.8, is_covered=True, name=None, matname=None)`
Then his script will fail.
What do you think?

Reply:
it's a clear spelling error. I don't think it's needed deprecation warning as soon as it's documented